blog/2015/11/swdcretreat2015-university_course.html

```diff
@@ -0,0 +1,436 @@
+<a href='http://www.nature.com/news/interactive-notebooks-sharing-the-code-1.16261'>Nature</a> and submission guidelines for
+<a href='http://journals.plos.org/plosone/s/materials-and-software-sharing'>PLOS ONE</a>
+have addressed <a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002205.html'>Greg's</a> initial observations that universites find that lab skills course is worth graduate credit.  The conversation contines
+<a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002207.html'>here</a>,
+<a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002210.html'>here</a>, and
+a blog post by Damien Irving <a href='http://software-carpentry.org/blog/2014/02/university-course.html'>here</a>.
+
+The Software Carpentry + Data Carpentry Instructor & Helper Retreat seemed like a good means to revisit the discussion.
+
+<h2>Who are we?</h2>
+<a href='https://twitter.com/chendaniely'>Daniel Chen</a> is a doctoral candidate in
+<a href='http://gbcb.vbi.vt.edu/gbcb/about_program'>Genetrics, Bioinformatics and Computational Biology (GBCB)</a>
+at Virginia Tech and currently working in the
+<a href='http://vbi.vt.edu/sdal'>Social and Decision Analytics Laboratory</a>
+under the <a href='http://vbi.vt.edu'>Virginia Bioinformatics Institute</a>.
+I received my MPH at the
+<a href='http://www.mailman.columbia.edu'>Mailman School of Public Health</a> in Epidemiology.
+
+Tiffany Timbers is ...
+
+Jenny Bryan is ...
+
+<h2>The Roundtable Discussion</h2>
+
+<p>
+  <h4>
+    Earlier this year, Anthony Scopatz and Kathryn Huff published <a href='http://physics.codes/'>Effective Computation in Physics</a> and created a course around the book.
+    Tiffany, can you talk about the <a href='https://phy3009.github.io/PHY3009-2015/'>course</a> your've been beta testing?
+  </h4>
+</p>
+
+<p>
+  [T]: Anthony has been teaching the course to Masters students and I've been building off Katy and Anthony's expereinces from teaching as well as my own from teaching Software Carpentry.
+  I decided to use this textbook as my course at Quest.
+  The book is based on Python, but many of the priinciples are language agnostic.
+  The book covers maby topics SWC teaches, such as Bash, Python, Make, documentation, licences, and many open science aspects.
+  Katy and Anthony also created a <a href='https://github.com/physics-codes/examples'>Github repo</a> of annotated ipython (jupyter) notebooks of the code in the book, as well as an <a href='https://github.com/physics-codes/seminar'>example syllabus</a>.
+  The notebook annotations are aimed to help those who are using the book as instructors and/or students.
+</p>
+
+<p>
+  <h4>
+    A reponse to my original discuss question revolved around administrative and university department politics on who should and who may teach a course like this.
```

Comment:
I'm not sure about the markup here.  You have `<p><h4>`, but this doesn't look like a header.  Is this [an][1] [aside][2]?  Is it just another paragraph?

[1]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside
[2]: http://html5doctor.com/aside-revisited/